apps/playwright/tests/search-workflows.spec.ts

```diff
@@ -0,0 +1,226 @@
+            details: "A Playwright test personal experience",
+            skills: {
+              sync: [
+                {
+                  details: `Test Skill ${technicalSkill.name.en}`,
+                  id: technicalSkill.id,
+                },
+              ],
+            },
+            startDate: FAR_PAST_DATE,
+            title: "Test Experience",
+          },
+        ],
+      },
+    });
+    await adminPage.addRolesToUser(createdUser.id, [
+      "guest",
+      "base_user",
+      "applicant",
+    ]);
+
+    const team = await getDCM();
+    const classifications = await getClassifications();
+    classification = classifications[0];
+
+    const adminUser = await adminPage.getMe();
+    // Accepted pool
+    const createdPool = await poolPage.createAndPublishPool({
+      userId: adminUser.id,
+      teamId: team.id,
+      classification,
+      skill: technicalSkill,
+      name: poolName,
+    });
+
+    const context = await browser.newContext();
+    const page = await context.newPage();
+    const applicationPage = new ApplicationPage(page, createdPool.id);
+    await loginBySub(applicationPage.page, sub);
+    const applicationUser: User = await applicationPage.getMe();
+    const application = await applicationPage.createGraphql(
+      createdUser.id,
+      applicationUser.experiences[0].id,
+    );
+    await applicationPage.submitGraphql(
+      application.id,
+      `${createdUser.firstName} signature`,
+    );
+
+    const adminContext = await browser.newContext();
+    const adminAppPage = await adminContext.newPage();
+    const adminApplicationPage = new ApplicationPage(
+      adminAppPage,
+      createdPool.id,
+    );
+    await loginBySub(adminAppPage, "admin@test.com");
+    await adminApplicationPage.updateStatusGraphql(
+      application.id,
+      PoolCandidateStatus.QualifiedAvailable,
+    );
+  });
+
+  test("Search and submit request", async ({ appPage }) => {
+    await appPage.page.goto("/en/search");
+    await appPage.waitForGraphqlResponse("SearchForm");
+
+    const poolCard = appPage.page.getByRole("article", {
+      name: new RegExp(poolName, "i"),
+    });
+
+    await expect(poolCard).toBeVisible();
+    await expect(poolCard).toContainText(/1 approximate match/i);
+
+    const classificationFilter = appPage.page.getByRole("combobox", {
+      name: /classification/i,
+    });
+
+    await classificationFilter.selectOption({ index: 2 });
+
+    await expectNoCandidate(appPage.page);
+
+    await classificationFilter.selectOption({
+      value: `${classification.group}-0${classification.level}`,
+    });
+
+    const streamFilter = appPage.page.getByRole("combobox", {
+      name: /stream/i,
+    });
+
+    await streamFilter.selectOption({ label: "Database Management" });
+
+    await expectNoCandidate(appPage.page);
+
+    await streamFilter.selectOption({
+      label: "Business Line Advisory Services",
+    });
+
+    await expect(poolCard).toBeVisible();
+
```

Comment:
No education requirement test?

Reply:
okay, I think I got all of the filters covered now with [add more filters to test coverage](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10560/commits/dfde102b82fac0c2f8d570f7675539768eb91a88)